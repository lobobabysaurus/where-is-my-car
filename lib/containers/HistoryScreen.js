import React from 'react'
import { Text, View } from 'react-native'

import Header, { HeaderButton } from '../components/Header'

const HistoryScreen = () => <View><Text>Bah</Text></View>

HistoryScreen.navigationOptions = ({navigation}) => {
  const mapButton =
    <HeaderButton onPress={() => navigation.goBack()}>
      Map
    </HeaderButton>
  return {
    header: <Header leftContent={mapButton} title="History"/>
  }
}

export default HistoryScreen
