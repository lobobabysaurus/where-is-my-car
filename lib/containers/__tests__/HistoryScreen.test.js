import React from 'react'
import renderer from 'react-test-renderer'

import HistoryScreen from '../HistoryScreen'

describe('HistoryScreen', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<HistoryScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
