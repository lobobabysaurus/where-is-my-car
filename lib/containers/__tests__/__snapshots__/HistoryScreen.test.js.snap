// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryScreen renders correctly 1`] = `
<View>
  <Text>
    Bah
  </Text>
</View>
`;
